packages/addresses/README.md

```diff
@@ -1,3 +1,26 @@
 # ZetaChain addresses
 
 This package includes the addresses and networks to use Zetachain.
+
+## Usage
+
+```js
+import { getAddress } from "@zetachain/addresses";
+
+const address = getAddress({address: "zetaToken", networkName: "goerli", zetaNetwork:"athens"});
+```
+
+## API
+
+
+| Method | Description |
+| :---- | ------ |
+| isTestnetNetworkName = (networkName: string) | Returns true if is a valid Testnet name |
+| isZetaTestnet = (networkName: string) | Returns true if is a valid ZetaTestnet name |
+| isMainnetNetworkName = (networkName: string) | Returns true if is a valid Mainnet name |
+| isNetworkName = (networkName: string) | Returns true if is a valid network name |
+| isZetaNetworkName = (networkName: string) | Returns true if is a valid Zeta network name |
```

Comment:
I'd add the return type here so it's clear that they also act as type assertions:
e.g.: `: networkName is TestnetNetworkName`

Reply:
I think it should be "if it's" instead of "if is"